README.md

```diff
@@ -54,34 +54,34 @@ $ make build
 To log into PagerDuty CLI use the command:
 
 ```
-pdcli login
+kite login
 ```
 This will prompt the user for an API key with necessary instructions of how to generate one. The API key will be saved for future use to the `~/.config/pagerduty-cli/config.json` file.
```

Comment:
should the config path be updated? i.e. ~/.config/kite vs ~/.config/pagerduty-cli

Reply:
It can be, updating now

Reply:
thanks for updating the source, the readme still has old reference tho

Reply:
Sorry about that, pushed the changes